semantic-conventions/src/opentelemetry/semconv/main.py

```diff
@@ -200,6 +201,12 @@ def add_md_parser(subparsers):
         required=False,
         action="store_true",
     )
+    parser.add_argument(
```

Comment:
Do you mean that when an attribute becomes frozen, we change it's brief to mention it there? It's a viable solution, but then it's  harder to read and review, harder to follow the same pattern, and easier to overlook.
It's also not possible to add any checks (if we want to).

I also think it could be useful to add a stability column and represent stability information separately from brief.